docs/guides/release_notes.md

```diff
@@ -0,0 +1,25 @@
+---
+page_title: "Release Notes"
+subcategory: "Guides"
+description: |-
+  <no value>
+---
+
+# Release Notes
+
+This document lists breaking changes and major features for each release of the Power Platform Terraform provider.  See the release notes for each version for more details.
+
+# v3.0.0
+
+## BREAKING CHANGES
+
+* `powerplatform_solution.solution_name` is no longer needed and has been removed.
+* `powerplatform_solution.settings_file_checksum` is now generated using SHA256 instead of MD5. This will cause a change in the checksum value of existing resources.
+* `powerplatform_solution.solution_file_checksum` is now generated using SHA256 instead of MD5. This will cause a change in the checksum value of existing resources.
+* `powerplatform_rest.expected_http_status` type is changed from []int64 to []int.  Practically, this should not affect any existing configurations.
+* `powerplatform_rest_query.expected_http_status` type is changed from []int64 to []int.  Practically, this should not affect any existing configurations.  
```

Comment:
if I remember correctly int is supported from not so long ago instead of int64, maybe we should look thou out the source code and replace all the occurrences? (Not sure how many we have)

Reply:
There are some other places that int might make more sense than int64 but don't know if it's a type mismatch between API and schema like it was here.  Would need further investigation that could be done in another PR.